client/src/session_state.rs

```diff
@@ -401,9 +407,16 @@ impl SessionState {
         };
         let response = self.send_request(request)?;
         if let SupportedMessage::OpenSecureChannelResponse(response) = response {
+            if self.ignore_clock_skew {
```

Comment:
Can you add a comment here to say what this is doing for clarity, i.e. to compensate for the difference in time between client and server

Reply:
Done (will push an additional commit with all requested changes in a bit)